src/Xrefcheck/Core.hs

```diff
@@ -102,26 +102,36 @@ data Anchor = Anchor
   deriving stock (Show, Eq, Generic)
   deriving anyclass NFData
 
+data CopyPaste = CopyPaste
+  { cpAnchorText :: Text
```

Comment:
I feel like `AnchorText` and `PlainText` sound confusing. [The issue](https://github.com/serokell/xrefcheck/issues/64) doesn't focus on anchors, but all type of links, external and internal. Perhaps, more generic `LinkText` and `URLText` would be better?